cms/envs/common.py

```diff
@@ -328,6 +328,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
     'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
     'ENABLE_ENROLLMENT_RESET': False,
+    'ENABLE_MOBILE_COURSE_AVAILABLE': False,
```

Comment:
If this feature flag is meant for disabling the functionality while `True` then it should be called `DISABLE_MOBILE_COURSE_AVAILABLE`.